NEWS

```diff
@@ -1,3 +1,10 @@
+CHANGES IN VERSION 1.25
+-----------------------
+
+NEW FEATURES
+
+    o Check for single colon typos and suggest double colons
```

Comment:
provide a bit more context here? I guess this involves pkg::foo() notation, and not some other use of `:`?